rooms/views.py

```diff
@@ -1,3 +1,60 @@
-from django.shortcuts import render
+import json, datetime
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Avg, Count
+
+from .models import *
+
+class RoomDetailView(View):
+    def get(self, request, room_id):
+        try:
+            room = Room.objects.get(id = room_id)
+            average_rating = room.review_set.aggregate(average = Avg('rating'))['average']
+
+            if average_rating == None:
+                average_rating = 0
```

Comment:
```suggestion
            if not average_rating:
                average_rating = 0
```
None 값자체가 boolean의 False을 타나내기 때문에 위와 같이 not 붙여서 사용을 많이 합니다.

Reply:
네엡! not을 이용하도록 하겠습니다! 😊